js/controllers/wizard/fileUpload/FileUploadWizardHandler.js

```diff
@@ -243,7 +258,8 @@
 				this.uploadedFile_.csrfToken = this.csrfToken_;
 				// Authorization policy expects to find the submissionFileId para
 				this.uploadedFile_.submissionFileId = this.uploadedFile_.id;
-				$.post(this.deleteUrl_, this.uploadedFile_,
```

Comment:
It looks like maybe `deleteUrl_` is no longer used?

Reply:
I think it's still used for file deletion